stdlib/jdbc/src/main/ballerina/src/java.jdbc/types.bal

```diff
@@ -232,7 +232,7 @@ public type UpdateResult record {|
 # + returnedError - The `Error` returned from the remote function in case of a failure
 public type BatchUpdateResult record {|
     int[] updatedRowCount;
-    map<anydata[]> generatedKeys;
+    map<anydata[]>? generatedKeys;
```

Comment:
Why only this map is mar as nillable? Why we are not being consistent with `UpdateResult` record?

Reply:
Fixed.